lua/oil/init.lua

```diff
@@ -393,7 +393,7 @@ M.open = function(dir)
 end
 
 ---Restore the buffer that was present when oil was opened
-M.close = function()
+M.close = function(exit_if_last_buf)
```

Comment:
Let's use an options table for this; it's more future-proof. And add a type annotation so that the auto-generated docs will show the parameter